pages/specification-links.md

```diff
@@ -432,8 +432,8 @@ The next unreleased draft is a work in progress.  You can [give feedback and get
 
 The specification links here link to the XML and JSON sources.  We do not provide rendered [work-in-progress](work-in-progress) drafts except near the very end of a publication cycle, during the final review period.
```

Comment:
Can you change "XML and JSON" to just state "raw", please?

Reply:
sure : )